ReportPortal.SpecFlowPlugin/EventArguments/InitializingEventArgs.cs

```diff
@@ -0,0 +1,17 @@
+﻿using ReportPortal.Client;
+using ReportPortal.SpecFlowPlugin.Configuration;
+
+namespace ReportPortal.SpecFlowPlugin.EventArguments
+{
+    public class InitializingEventArgs
+    {
+        public InitializingEventArgs(Server server)
```

Comment:
Let's keep `Config` object here instead of `Server`. User will be able to override config at all.

Reply:
Implemented.